outputs.tf

```
@@ -30,10 +30,12 @@ output "ecr_dkr_domain_name" {
 
 locals {
   s3_no_proxy = contains(var.aws_vpce_services, "s3") ? [".s3.${var.region}.amazonaws.com"] : []
+  ecr_api_no_proxy = contains(var.aws_vpce_services, "ecr.api") ? ["api.ecr.${var.region}.amazonaws.com"] : []
+  ecr_dkr_no_proxy = contains(var.aws_vpce_services, "ecr.dkr") ? ["dkr.ecr.${var.region}.amazonaws.com"] : []
```

```suggestion
  ecr_dkr_no_proxy = contains(var.aws_vpce_services, "ecr.dkr") ? [".dkr.ecr.${var.region}.amazonaws.com"] : []
```

I think containers are pulled down in a virtual-hosted addressing style, e.g. `containerhash.dkr.ecr.blah.amazonaws.com` so wildcarding it here should allow all containers to be pulled down.